source/sync/data-model/update-data-model/update-data-model-development.txt

```diff
@@ -0,0 +1,231 @@
+.. _update-data-model-development-mode:
+
+==================================
+Update a Data Model in Development
+==================================
+
+.. meta::
+   :description: Use Device Sync's Development Mode feature to update your data model while developing your client app using Atlas Device SDK.
+   :keywords: Realm, object model, C++ SDK, Flutter SDK, Java SDK, Kotlin SDK, .NET SDK, Node.js SDK, React Native SDK, Swift SDK
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+After you :ref:`define your data model <create-sync-schema>`, you can continue
+to use Device Sync's :ref:`Development Mode <development-mode>` feature to
+streamline the process of updating your data model as you develop your app.
+
+When Development Mode is enabled, you can work entirely from your client code
+and define all of your objects in your language of choice. As you make changes
+to your object models and connect to the server, the server automatically
+applies those changes to the corresponding server-side JSON schema.
+
+This page describes how to update your data model while Development Mode is
+enabled. To learn how to update your data model after your app goes to
+production, refer to :ref:`update-data-model-production`.
+
+.. include:: /includes/device-sync/important-disable-dev-mode-production.rst
+
+.. _dev-mode-breaking-changes:
+
+Development Mode
+----------------
+
+**Development Mode** is a Device Sync configuration setting that lets you
+define and modify your SDK data model directly in your client code, and App
+Services handles the server-side updates.
+
+Each time a client connects and syncs data with an updated data model while
+Development Mode is enabled, App Services applies your changes to the
+server-side schema. This ensures that your data model remains consistent as you
+develop your app.
+
+For more information on Development Mode and how to enable it in your app, refer
+to :ref:`development-mode`.
+
+Non-Breaking Changes
+~~~~~~~~~~~~~~~~~~~~
+
+When you make :ref:`non-breaking changes <non-breaking-changes-overview>` to
+your client-side model while Development Mode is enabled, the next time you
+connect a client and sync data, App Services automatically updates the
+corresponding schema.
+
+Although you can make non-breaking changes to your client-side data model or to
+your server-side schema, we recommend updating in the client while you are
+developing your app. This is because some changes are only considered
+non-breaking when made from the client side. If you do choose to make the
+changes to your server-side schema, you must manually update your client-side
+objects.
+
+The following changes to your client-side model are considered non-breaking:
+
+- Adding a new object type
+- Adding a new property
+- Adding a default value on an existing property
+- Removing an object type
```

Comment:
This and the `removing a property` change will be considered breaking when made in development mode because they are breaking changes on the server. They are non-breaking when the client is _not_ in development mode.

Reply:
Ahhh, okay. That makes sense. Thanks for catching!